controllers/main_controller.rb

```diff
@@ -9,3 +9,19 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/about" do
+  erb :"main/about"
+end
+
+MyApp.get "/favorites" do
+  erb :"main/favorites"
+end
+
+MyApp.get "/greet/Sumeet" do
```

Comment:
What happens if you remove this controller action:

```ruby
MyApp.get "/greet/Sumeet" do
  erb :"main/greet/Sumeet"
end
```

Do your tests continue to pass? How about going to http://localhost:9292/greet/Sumeet in your browser manually? If things still work, why?

tias, please.